$NetBSD: patch-ar,v 1.1 2009/03/20 08:50:51 hasso Exp $

More portable way to print st_size which is off_t.

--- lib/packagekit-glib/pk-service-pack.c.orig	2009-03-20 10:14:37 +0200
+++ lib/packagekit-glib/pk-service-pack.c	2009-03-20 10:15:18 +0200
@@ -599,7 +599,7 @@ pk_service_pack_archive_add_file (struct
 				      "file not found %s", filename);
 		goto out;
 	}
-	egg_debug ("stat(%s), size=%lu bytes\n", filename, st.st_size);
+	egg_debug ("stat(%s), size=%jd bytes\n", filename, (intmax_t)st.st_size);
 
 	/* create new entry */
 	entry = archive_entry_new ();
